views/js/controller/Publish/index.js

```diff
@@ -95,6 +95,9 @@ define([
                 refreshTree(testUri);
             })
             .on('error', function(err){
+                if (err && err.errorMsg) {
```

Comment:
please ask a FE Engineer to validate this  :)

Reply:
> `index.js` change approved.

- @oatymart 

;)

Reply:
Already approved ✔️